### PR TITLE
BUG: Don't use mutable default values

### PR DESCRIPTION
**Pull request by bashtage**
Do not use mutable default values

### CODE REVIEW

**Comment by jbrockmendel** [comment]
very much +1

File: statsmodels/robust/scale.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 from . import norms
 from statsmodels.tools import tools
 
-def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
+
+def mad(a, c=None, axis=0, center=np.median):
```
Comment (josef-pkt) [inline statsmodels/robust/scale.py]: Gaussian.ppf(3/4.) is a float scalar and not mutable
?

**Comment by josef-pkt** [comment]
AFAIU: one too much, the other ones look goot

File: statsmodels/robust/scale.py (hunk 1 above)
Comment (jbrockmendel) [reply statsmodels/robust/scale.py]: @bashtage josef is right on this one.  If reverting that will get this otherwise-solid PR through, go for it.

**Comment by bashtage** [comment]
Reverted

**Comment by josef-pkt** [comment]
(we have one unrelated 32-bit windows problem)

LGTM now, merging

@bashtage Thank you

**Pull request merged**